packages/docusaurus/bin/beforeCli.mjs

```diff
@@ -130,9 +130,9 @@ export default async function beforeCli() {
       if (!yarnVersion) {
         return `npm i ${siteDocusaurusPackagesForUpdate}`;
       }
-      return yarnVersion === 1
-        ? `yarn upgrade ${siteDocusaurusPackagesForUpdate}`
-        : `yarn up ${siteDocusaurusPackagesForUpdate}`;
+      return yarnVersion >= 2
+        ? `yarn up ${siteDocusaurusPackagesForUpdate}`
+        : `yarn upgrade ${siteDocusaurusPackagesForUpdate}`;
```

Comment:
Modified the conditional in the `getUpgradeCommand()` method to `yarnVersion >= 2`. I think this makes more sense because it's different for Yarn 2+.